src/plugins/news/screens/article/schemas.js

```diff
@@ -2,6 +2,8 @@ import ArticleV1Mixin from '@triniti/schemas/triniti/news/mixin/article/ArticleV
 import SearchArticlesRequestV1Mixin from '@triniti/schemas/triniti/news/mixin/search-articles-request/SearchArticlesRequestV1Mixin';
 import resolveSchema from '@triniti/cms/utils/resolveSchema';
 
+import NodeLabelsUpdatedV1 from '@gdbots/schemas/gdbots/ncr/event/NodeLabelsUpdatedV1';
```

Comment:
@rsumilang not needed, use the static ref in the labels-form itself.  since it's concrete no mixin resolution is needed.